### ISSUE TITLE
Using a custom manifest downgrades the android version

### ISSUE THREAD

**Issue by hkparker** [fyne-io/fyne#5220]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [X] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

After creating an android app with `fyne package` and installing with `adb install`, I changed the application to use a custom `AndroidManifest.xml`, which I created by extracting the on generated by fyne and modifying the permissions (I also had to remove the icon declaration because of a fyne error, but that's not relevant).

Upon building this app with the updated manifest, using the same `fyne package` command, and attempting to install, `adb` complained that 

`adb: failed to install App.apk: Failure [INSTALL_FAILED_VERSION_DOWNGRADE: Downgrade detected: Update version code 0 is older than current 1]`

This is east to work around with `adb install -r -d App.apk`, but it's weird that it happened, I don't see anything in the manifest that suggestions I'm now on version `0`.

### How to reproduce

1. Create an android app with `fyne package`, install with `adb install`
2. Add a custom `AndroidManifest.xml` to the repository and rebuild
3. Try to install

### Example code

I don't believe there's any code relevant to reproducing this issue

### Fyne version

2.5.2

### Go compiler version

go1.23.2 linux/amd64

### Operating system and version

Arch Linux / Android 14

**Comment by andydotxyz** [comment]
You'd be looking for the key `android:versionCode`

**Comment by hkparker** [comment]
Huh, yeah that was missing.  However I got my android manifest by using `apktool` to extract it from the apk Fyne built originally.  I wonder what step in the process dropped that field, I assumed I was looking at the actual manifest `fyne package` created when I first built the app, but maybe this is an issue with `apktool` and it stripped down the manifest for some reason.  I'll close this out since it seems likely unrelated to Fyne.
